src/pages/index.js

```diff
@@ -1,45 +1,57 @@
 import "./index.css";
-import { enableValidation, settings } from "../scripts/validation.js";
+import {
+  enableValidation,
+  settings,
+  resetValidation,
+  disableButton,
+} from "../scripts/validation.js";
+import { setButtonText } from "../utils/helpers.js";
+
 import logo from "../images/logo.svg";
 import avatar from "../images/avatar.jpg";
 import penIcon from "../images/pen.svg";
+import pencilIcon from "../images/pen_light.svg";
 import plusIcon from "../images/plus_sign.svg";
 
+import Api from "../utils/Api.js";
+
+
 document.querySelector(".header__logo").src = logo;
 document.querySelector(".profile__avatar").src = avatar;
+document.querySelector(".profile__pencil-icon").src = pencilIcon;
 document.querySelector(".profile__edit-btn img").src = penIcon;
 document.querySelector(".profile__add-btn img").src = plusIcon;
 
+document.querySelector(".footer__copyrights").textContent = 
+  `${new Date().getFullYear()} © Spots`;
 
-const initialCards = [
-  {
-    name: "Val Thorens",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/1-photo-by-moritz-feldmann-from-pexels.jpg",
-  },
-  {
-    name: "Restaurant terrace",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/2-photo-by-ceiline-from-pexels.jpg",
-  },
-  {
-    name: "An outdoor cafe",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/3-photo-by-tubanur-dogan-from-pexels.jpg",
-  },
-  {
-    name: "A very long bridge, over the forest and through the trees",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/4-photo-by-maurice-laschet-from-pexels.jpg",
-  },
-  {
-    name: "Tunnel with morning light",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/5-photo-by-van-anh-nguyen-from-pexels.jpg",
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "15d20bab-3906-48d9-b03a-14f860add378",
+    "Content-Type": "application/json",
   },
-  {
-    name: "Mountain house",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/6-photo-by-moritz-feldmann-from-pexels.jpg",
-  },
-];
+});
+
+const cardsList = document.querySelector(".cards__list");
+
+api
+  .getAppInfo()
+  .then(([cards, userInfo]) => {
+    cards.forEach((item) => {
+      const cardElement = getCardElement(item);
+      cardsList.append(cardElement);
+    });
+    document.querySelector(".profile__avatar").src = userInfo.avatar;
+    document.querySelector(".profile__name").textContent = userInfo.name;
+    document.querySelector(".profile__description").textContent =
```

Comment:
`document.querySelector(".profile__avatar")`
`document.querySelector(".profile__name")`
`document.querySelector(".profile__description")`

It’s better to find **constant DOM elements** only 1 time at the top of the file so as not to waste resources on searching them again and again when you call a method (function) because searching is a very **hard** operation for the browser engine


By the way, you've already found `profileName` and `profileDescription` in lines 55-56